cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -626,4 +633,166 @@ define([
             });
         });
     });
+
+    describe('GroupList', function() {
```

Comment:
Maybe call this ```GroupListView``` to make it clearer that it isn't testing a list of groups but a view that can show a list.